#20 위키 입력 테스트코드 작성

kmmin78 opened the pull request:
주요 내용
1. PostEntity, Service, Repository 작성
2. Post 관련 UserEntity 작성
3. 본문 100자 이상, 내용 10,000자 이상 시 실패하는 Test Case 작성

Commit:
test

kmmin78 committed:
Merge pull request #1 from kmmin78/feature/001

test

Commit:
test2

Commit:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	app-main/src/main/kotlin/kr/flab/wiki/app/WikiApplication.kt

Commit:
test2

Commit:
test3

Commit:
1. executable jar build
2. apply detekt

Commit:
1. executable jar build
2. apply detekt

Commit:
1. executable jar build
2. apply detekt

Commit:
git.gradle 추가

Commit:
dependencies 추가

Commit:
dependencies 추가

Commit:
dependencies 추가

Commit:
beta profile added in develop branch

Commit:
git test

Commit:
0.0.5

Commit:
0.0.6

Commit:
0.0.6
End

Commit:
0.0.61
tdd

Commit:
0.0.61
tdd

Commit:
0.0.7
1. tdd practice
2. mockito + hamcrest
3. junit nested test

Commit:
#14 자바코드 코틀린 전환

1. 자바코드 삭제
2. 코틀린으로 재작성
3. 기존 자바코드와 동일하게 동작하는 것 까지 확인

Commit:
#15 코드리뷰 피드백 반영

주요 내용
1. 싱글턴(object)로 작성
2. User Validation을 UserEntity로 delegate
    -> UserLoginServiceImpl에서 로직 제거
    -> UserEntity에 custom setter 추가
    -> UserEntity setter 테스트는 어디서 해야 될까?

Commit:
#20 위키 입력 테스트코드 작성

주요 내용
1. 테스트 유틸클래스 명칭 변경
2. PostServiceTest 클래스 기본 골조 추가
3. PostEntity, Service, Repository 작성
4. Post 관련 UserEntity 작성
5. 본문 100자 이상, 내용 10,000자 이상 시 실패하는 Test Case 작성
6. title 중복 등록 시 exception 발생하는 Test Case 작성

FrancescoJo commented:
이거 File changes 가 너무 많습니다. 작업을 다시 작은 단위로 나눠서 PR 을 여러개로 올려 주시기 바랍니다.

1 PR 당 권장 File changes: 15 - 20

Commit:
Merge remote-tracking branch 'upstream/develop' into develop

Commit:
#20 위키 입력 테스트코드 작성

주요 내용
1. 테스트 유틸클래스 명칭 변경
2. PostServiceTest 클래스 기본 골조 추가
3. PostEntity, Service, Repository 작성
4. Post 관련 UserEntity 작성
5. 본문 100자 이상, 내용 10,000자 이상 시 실패하는 Test Case 작성
6. title 중복 등록 시 exception 발생하는 Test Case 작성

Commit:
Merge branch 'feature/kms/003' of https://github.com/kmmin78/awesome-wiki into feature/kms/003

Commit:
주요 내용
1. 포스트 조회 함수 추가
2. database 객체 repository에 주입
3. 포스트 조회 테스트 케이스 추가
